client/src/components/posting-page/JobPosting.js

```diff
@@ -182,10 +184,20 @@ const ErrorMessage = ({ errorMessage }) => {
   )
 }
 
-const HiddenNotification = ({ jobPosting }) => {
+const HiddenNotification = ({ showFrom, showTo }) => {
+  let visibilityMessage = 'Permanently hidden'
+
+  if (showFrom) {
+    visibilityMessage = `Visible from ${showFrom}`
+
+    if (showTo) {
+      visibilityMessage = `Visible from ${showFrom} to ${showTo}`
```

Comment:
Now the dates are shown in yyyy-mm-dd format. However I think the dates would be easier to read if they were displayed like 12(th) December 2018 or 12.12.2018